test/helpers/fixture_flows_helper.rb

```diff
@@ -1,4 +1,16 @@
 module FixtureFlowsHelper
+  BUTTON_TEXT_AND_LINK_FIXTURE = {
+    "overseas-passports": {
+      start_button_text: "Continue",
+      href: "https://www.passport.service.gov.uk/filter"
+    },
+    "calculate-your-child-maintenance": {
+      start_button_text: "Calculate your child maintenance",
+      href: "/calculate-your-child-maintenance/y"
+    }
+  }.with_indifferent_access
```

Comment:
If we add `with_indifferent_access` to a hash that is set up in a test, it has to be because it mimics the data in the code we are testing. I don't think this is happening here. 

It would be better to use [deep_stringify_keys][1] or [deep_symbolize_keys][2].

[1]: https://github.com/rails/rails/blob/8b2d978d2892abe85859c75dfcbdfe218b98de81/activesupport/lib/active_support/hash_with_indifferent_access.rb#L260
[2]: https://github.com/rails/rails/blob/8b2d978d2892abe85859c75dfcbdfe218b98de81/activesupport/lib/active_support/hash_with_indifferent_access.rb#L264